src/js/module/component/button.js

```diff
@@ -1,6 +1,5 @@
-import { View } from '../../core';
 import Template from './button.hbs';
-import { Model } from 'backbone';
+import { Model, View } from 'backbone';
```

Comment:
We don't need reference to app object in so atomic component. And inheritance from Core.View is complicating usage of this class.